File: Arithmetic_HCF.vb
Module Arithmetic_HCF

    Sub Main()

        ' Use the HCF Class
        Dim group As New List(Of Integer)({20, 30, 40})
        Dim hcf As New HCF
        hcf._init_(group)
        Dim answer = hcf.getHCF()

        Console.Write("The H.C.F. of " & String.Join(", ", group) & " is " & answer)

    End Sub

End Module

File: Arithmetic_HCF.pl
#!/usr/bin/perl;
use strict;
use warnings;
use HCF;

# Useful variables
my $answer;
my @set;

# Use the HCF module/class
@set = (20, 30, 40);
my $hcf = HCF->new(\@set);
$answer = $hcf->getHCF();
print ("The H.C.F. of ", join(", ", @set), " is $answer\n");


print "\n\n";